demo/index.html

```diff
@@ -231,16 +219,16 @@ <h3>Iron-form respects the <code>novalidate</code> form attribute</h3>
             <input name="animal" placeholder="animal" required="" value="meerkat"><br>
             <paper-checkbox name="cheese" required="">Cheese</paper-checkbox>
             <br>
-            <paper-button raised="" onclick="form5.submit()">Submit</paper-button>
```

Comment:
Where'd all this stuff go?

Reply:
right below. this is modulizer doing ugly to boolean attributes (it should be `raised`, not `raised=""`)